src/app/static/src/app/components/genericChart/GenericChart.vue

```diff
@@ -60,12 +60,12 @@
                 </div>
                 <div v-for="dataSource in chartConfigValues.dataSourceConfigValues.filter(ds => ds.tableConfig)"
                      :key="dataSource.config.id">
-                    <download-indicator :filtered-data="filteredDataWithoutPages ? filteredDataWithoutPages[dataSource.config.id] : null"
-                                        :unfiltered-data="unfilteredData ? unfilteredData[dataSource.config.id] : []"></download-indicator>
-                    <generic-chart-table :table-config="dataSource.tableConfig || { columns: [] }"
+                    <download-indicator :filtered-data="filteredDataWithoutPages![dataSource.config.id]"
```

Comment:
so in master there is no non null assertion but this may cause a run time vue props warning, I can remove these but console might then contain some more warnings